### DISCUSSION TITLE
TSM doesnt restore a FF-session anymore

### DISCUSSION THREAD

**Discussion by LittleBit2024** [sienori/Tab-Session-Manager#1332] [Q&A]
The Firefox AddOn 'Tab Session Manager' (TSM) worked nice
but recently TSM refused to restore a session. It just doesnt work any longer.
When I create a Window manually, I can restore the Tab's using TSM... although this requires me to redo the (un)pin-, container- and title- settings.
Im using the most recent (v6.12.2) version with FF v126 on a Win10 laptop.

Reading this post, the problem already existed 2 years ago..
  https://forums.tomshardware.com/threads/tab-session-manager-refuses-to-restore-sessions-in-firefox.3782153/

I have no idea about how to solve it. Any suggestions ?

**Comment by LittleBit2024** [comment]
And to be more precise:
- it is NOT restoring a session
- it is NOT restoring a Window
- but it is still restoring a Tab

**Comment by awprawpr** [comment]
Does the lack of replies imply the developer has deserted the app?
